docs/DeveloperGuide.md

```diff
@@ -194,20 +194,27 @@ The sequence diagram below illustrates the creation and execution of a `CsvExpor
 
 * **Alternative 1 (Current Choice):** Always save the CSV file to a fixed, pre-defined location without user input.
   * Pros:
-        1. Simplifies the command's implementation by removing the need to parse and validate user-provided file paths.
+      1. Simplifies the command's implementation by removing the need to parse and validate user-provided file paths.
   * Cons:
-        1. Reduces user flexibility in determining where the CSV file should be saved.
+      1. Reduces user flexibility in determining where the CSV file should be saved.
 * **Alternative 2:** Allow users to specify a file path, defaulting to a pre-defined location if not specified.
   * Pros:
-        1. Provides flexibility for users to save the CSV file wherever they prefer.
+      1. Provides flexibility for users to save the CSV file wherever they prefer.
   * Cons:
-        1. Additional error handling is required to manage invalid or inaccessible file paths.
+      1. Additional error handling is required to manage invalid or inaccessible file paths.
 
```

Comment:
Thanks for fixing the formatting! Actually if its only one thing, we can omit the list I guess..

Reply:
Good points. Updated, thanks!